rs/replicated_state/src/canister_snapshots.rs

```diff
@@ -283,6 +286,7 @@ pub struct CanisterSnapshot {
     /// The certified data blob belonging to the canister.
     certified_data: Vec<u8>,
     /// Snapshot of chunked store.
+    #[validate_eq(Recursive)]
     chunk_store: WasmChunkStore,
     execution_snapshot: ExecutionStateSnapshot,
```

Comment:
This needs to be recursive, as there are PageMaps inside.

Reply:
I'd expect it's recursive by default, unless we're explicitly opting out with `Skip`...